README.md

```diff
@@ -99,6 +99,34 @@ of an Eclipse Hono instance as shown in the diagram below.
    back end component which receives vehicle data via Hono's north bound Kafka based Telemetry API and writes it to the
    Influx DB.
 
+
+# Using Zenoh Router to geographically distribute the vehicle data
+
+The integration Extend FMS Forwarder to publish data in Zenoh and configure Zenoh Router to also receive data over MQTT:
+
+- Assess the current FMS Forwarder and identify the necessary modifications to enable data publication through Zenoh.
+- Implement the required code changes to publish data via Zenoh.
+- MQTT plugin configuration on the Zenoh Router to as well support MQTT interfaces.
+
+<img src="img/architecture-zenoh.drawio.png">
+
+We now provide the instructions on how you can test the communication using Zenoh
+
+1. Create a local docker image for FMS Forworder to support the zenoh communication, run the below command from  fleet-management/components/ directory:
+   
+   ```sh
+   docker build -t fms-forwarder:custom -f Dockerfile.fms-forwarder .
+   ```
+   
+2. Start up the zenoh router and back end services using Docker Compose files in the top level (fleet-management/) folder:
+
+   ```sh
+   docker compose -f ./fms-blueprint-compose.yaml -f ./fms-blueprint-compose-zenoh.yaml up --detach
+   ```
+
+Here you find the [Zenoh based Python Subscriber](https://github.com/vivekpandey02/fms-zenoh-test-subscriber) application to validate the data transmissted by FMS Fowarder.
```

Comment:
Can't we simply include the necessary code from the other repo in this PR as well?

Reply:
Readme updated for as per latest zenoh communication architectured.